note-end

```diff
@@ -1,38 +1,37 @@
 #!/bin/bash
 
-required_date="${1:-today}"
+appRootFolder="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
As a general rule, I want to make a difference in the code between the folders used to reference the app and the note storage location. What do you think about my approach ? 
appRootFolder for the application code and notesRootFolder for the folder used to store the notes.